sdk/textanalytics/Azure.AI.TextAnalytics/src/PageableOperation.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+
+#nullable enable
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Represents a pageable long-running operation that exposes the results
+    /// in either synchronous or asynchronous format.
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public abstract class PageableOperation<T> : Operation<AsyncPageable<T>> where T : notnull
+    {
+        /// <summary>
+        /// Gets the final result of the long-running operation in a synchronous way.
+        /// </summary>
+        /// <remarks>
+        /// This property can be accessed only after the operation completes successfully (HasValue is true).
+        /// </remarks>
+        public override AsyncPageable<T> Value => GetValuesAsync();
+
+        /// <summary>
+        /// Gets the final result of the long-running operation in an asynchronous way.
+        /// </summary>
+        /// <remarks>
+        /// Operation must complete successfully (HasValue is true) for it to provide values.
+        /// </remarks>
+        public abstract AsyncPageable<T> GetValuesAsync();
+
+        /// <summary>
+        /// Gets the final result of the long-running operation in an asynchronous way.
+        /// </summary>
+        /// <remarks>
+        /// Operation must complete successfully (HasValue is true) for it to provide values.
+        /// </remarks>
+        public abstract Pageable<T> GetValues();
+
+        /// <inheritdoc />
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override bool Equals(object? obj) => base.Equals(obj);
```

Comment:
Don't we hide these in the `Operation<T>`?

Reply:
yeah u are right. we don't need them here. Will delete